### DISCUSSION TITLE
Graphs not loading correctly on mobile

### DISCUSSION THREAD

**Discussion by armand200011** [apexcharts/apexcharts.js#4250] [General]
I have been experiencing that the apex charts does not load correctly in the project that I am working on certain mobile devices I have test on two different phone models and on both of them the issue occurred firstly on Xaomi Redmi Note 11 Pro and on a Samsung Galaxy A34. 
I tested now again on the apexchart.js site on both chrome and fire fox browsers. The graphs were still getting duplicated sometimes and not functioning correctly on chrome, compared to fire fox browser where I opened that same page but everything functions properly.

Below were my findings when I opened the site on both browsers.
Chrome link:
https://drive.google.com/file/d/1MS1MBO0PP0hrlc-mtRukhqIBDwRzOssb/view?usp=sharing
Fire Fox link:
 https://drive.google.com/file/d/1gPIAHEsQZKGYFkA5A3uUh1Ot_ccfY-0h/view?usp=sharing

_Originally posted by @armand200011 in https://github.com/apexcharts/Blazor-ApexCharts/issues/405#issuecomment-1945760805_

**Comment by 07Sam** [comment]
I am having similar issues... Please let me know if you find a solution to this

**Comment by jarodLesser** [comment]
I am experiencing this same issue. It seems to work okay on PC, just the phone where it is giving me issues

**Comment by Lennert0d** [comment]
I have also issue happen on my phone.

**Comment by Zapnologica** [comment]
Is there any progress with regards to this bug?

We are seeing this behavior as well. But only on certain mobile mobile phones,
